src/CoreMedia/CMSampleBuffer.cs

```diff
@@ -834,6 +841,72 @@ public CMSampleBufferError CopyPCMDataIntoAudioBufferList (int frameOffset, int
 
 			return new CMSampleBuffer (buffer, true);
 		}
+
+		[iOS (12,2)]
+		[TV (12,2)]
+		[Mac (10,14,4, onlyOn64: true)]
+		[DllImport (Constants.CoreMediaLibrary)]
+		static unsafe extern /* OSStatus */ CMSampleBufferError CMSampleBufferCreateForImageBufferWithMakeDataReadyHandler (
```

Comment:
`CMSampleBufferCreateForImageBufferWithMakeDataReadyHandler` is the block based version of `CMSampleBufferCreateForImageBuffer`, which we've already bound: https://github.com/xamarin/xamarin-macios/blob/8e321095063c2b5e3370b39c78bdbc373e7be5f3/src/CoreMedia/CMSampleBuffer.cs#L237-L268

Reply:
good catch, I missed it :(
better remove it